dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -120,6 +120,29 @@ public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceC
         application.run("db", "migrate", configPath, "--include", "1.5.0");
     }
 
+    /**
+     * Drops and recreates database from migrations for test confidential 1 without VerificationInformation Migration
+     * @param support reference to testing instance of the dockstore web service
+     * @throws Exception
+     */
+    public static void cleanStatePrivate1NoVerificationMigration(DropwizardTestSupport<DockstoreWebserviceConfiguration> support) throws Exception {
```

Comment:
cleanStatePrivate2 vs cleanStatePrivate1 was probably a mistake and this takes it even further. 
If you need to control behavior, just use new parameters and use parameter overloading so that the existing signature maintains the old behaviour. 

i.e. the real work could be in 
`cleanStatePrivate(boolean confidential1, enum stopPoint, etc.`

cleanStatePrivate1 would call cleanStatePrivate(true, 1.5.0, etc.) 
cleanStatePrivate2 would call cleanStatePrivate(false, 1.5.0, etc.) 
Your new method would call cleanStatePrivate(false, 1.4.0, etc.)  and then cleanStatePrivate(false, 1.5.0, etc.)  to finish up